build/.night-build.yml

```diff
@@ -18,6 +18,7 @@ schedules:
     - releases/1.6.0
     - features/automl
     - features/integrationPackage
+    - feature/*
```

Comment:
Maybe we should remove those other explicit features, and maybe we should add a release wildcard as well.  What do you think @michaelgsharp?

Reply:
I thought about it and figured it would come up in the PR :).

The other ones are "features" but I'm not sure if we still need them?


+1 on releases!

Reply:
I think it should be release/*.  Probably this is somewhat stale.

Reply:
I agree -  happy to change it but all the old branches are - releases/*. Should we just leave as-is?

Reply:
Old branches are effectively dead now and won’t be building any more.  Our servicing policy is roll forward to latest